### DISCUSSION TITLE
Getting build errors on 3party boost files

### DISCUSSION THREAD

**Discussion by softreigns** [organicmaps/organicmaps#1764] [Q&A] [answered]
#include <boost/range.hpp>, could not resolve this reference, file not found.

also same with 
#include <boost/utility.hpp>
#include <boost/range.hpp>
#include <boost/function.hpp>
#include <boost/lambda/bind.hpp>
#include <boost/lambda/construct.hpp>

am i missing any pre build, project setting ?

**Comment by biodranik** [comment, marked as answer]
Did you clone all submodules and run ./configure.sh ?

**Reply by softreigns** [reply]
no, i thought (./configure.sh) is only if just to do a build without having to run in xcode. Will try that.

**Reply by softreigns** [reply]
worked. thanks !!

**Reply by ssvas9** [reply]
I am facing same issue ... Running the Android Studio on Windows. What should I do? and where can i fid this instructions to build?
[org_maps_build_issue.txt](https://github.com/user-attachments/files/22092764/org_maps_build_issue.txt)
